### PR TITLE
do not merge - this is a concept - feat: acquisition checkpoint

### PR DESCRIPTION
**Pull request by ekremney**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -269,3 +271,48 @@ fflags.enabled('email', () => {
...
+    owned: ['owned'],
+    qr: ['qr', 'qrcode'],
+    search: ['direct', 'google', 'googleflights', 'paidsearch', 'paidsearchnb', 'sea', 'sem'],
+    sms: ['sms'],
+    social: ['facebook', 'gnews', 'instagramfeed', 'instagramreels', 'instagramstories', 'line', 'linkedin', 'metasearch', 'organicsocialown', 'paidsocial', 'social', 'sociallinkedin', 'socialpaid'],
+    video: ['native', 'paidvideo', 'pvid', 'video', 'youtube'],
+    web: ['webapp'],
+  };
+  const sources = {
+    paid: ['affiliate', 'audio', 'display', 'local', 'search', 'social', 'video'],
```
[the remark this answers is not in the document]
Comment (trieloff) [reply src/index.js:297]: I don't see enough values to justify this. The most common I see is `igshopping` with 4 hosts, but that would be `instagram`.

File: src/index.js [hunk 2]
```diff
@@ -269,3 +271,48 @@ fflags.enabled('email', () => {
...
+    { regex: /face|fb|meta/i, result: 'facebook' },
+    { regex: /email/i, result: 'email' },
+    { regex: /bing/i, result: 'bing' },
+    { regex: /amazon|ctv/i, result: 'amazon' },
+    { regex: /qr/i, result: 'qrcode' },
+    { regex: /youtube|yt/i, result: 'youtube' },
+  ];
+  const utmMedium = sanitize(new URLSearchParams(window.location.search).get('utm_medium'));
+  const utmSource = sanitize(new URLSearchParams(window.location.search).get('utm_source'));
+  const preVendor = vendorMappings.find(({ regex }) => regex.test(utmSource))?.result || utmSource;
```
Comment (trieloff) [inline src/index.js:313]: The optional chaining limits browser compatibility. I don't think we use it anywhere else in the code base.

**Review by trieloff — CHANGES_REQUESTED** [review]
Try to run the same tests that https://github.com/adobe/helix-rum-collector/pull/347 has, so that we can identify the differences in approach.

**Comment by trieloff** [comment]
Also, this should go against `main` – no new features in `1.x` we said.

File: src/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramboz) [reply src/index.js:297]: @trieloff Shopping is not an actual value you find, it's a category you can derive from the hostname, same as social in a sense or search. Shopping is usually used for e-commerce sites. So think big websites like Amazon/Etsy/Alibaba/Rakuten/…, or anything based on Shopify/Magento/Squarespace/…

**Pull request closed without merging**